hls4ml/templates/vivado/nnet_utils/nnet_conv1d.h

```diff
@@ -53,9 +53,19 @@ void pointwise_conv_1d_cl(data_T data[CONFIG_T::in_width * CONFIG_T::n_chan],
 
     #pragma HLS INLINE region
 
-    // Nothing special to be done for io_parallel implementation
     if (CONFIG_T::strategy == nnet::latency) {
-        conv_1d_latency_cl<data_T, res_T, CONFIG_T>(data, res, weights, biases);
+        if (CONFIG_T::implementation == conv_implementation::pointwise) {
+            // Use pointwise unrolled implementation
+            if (CONFIG_T::reuse_factor > 1) {
+                CONFIG_T::template pointwise_conv<data_T, res_T, CONFIG_T>::pointwise_conv(data, res, weights, biases);
```

Comment:
So a model with a kernel > 1 but set conv_implementation to pointwise, the code will be generated and since it doesn't have assert it will proceed to do an incorrect computation? It's nonsense to us, but I fear that misunderstanding of the docs or just bugs in conversion scripts may trigger this

Reply:
So there is an assert here: 
https://github.com/fastmachinelearning/hls4ml/blob/1dd2603558f8ceb6d16b449c67e52567650d3eaf/hls4ml/templates/vivado/nnet_utils/nnet_conv1d_latency.h#L88-L92

Where else should we add it?

Reply:
resolved i think with the latest commits.